### ISSUE TITLE
Shazbat.tv Configuration failed: The username and password entered do not match

### ISSUE THREAD

**Issue by bodenkers** [Jackett/Jackett#1090]
After upgrading to Jackett Version 0.7.1043.0 my shazbat.tv tracker is no longer working. I am getting the following error message.

Shazbat.tv Configuration failed: The username and password entered do not match

I have verified I am supplying the correct username/password.

I have enabled advanced logging and below is the output. I have masked my username and password from the output.

2017-02-20 19:36:47.6658 Info   http://tower:9117/Admin/configure_indexer
2017-02-20 19:36:47.6664 Info DefaultHttpControllerSelector SelectController Route='MS_SubRoutes:System.Web.Http.Routing.IHttpRouteData[]'
2017-02-20 19:36:47.6664 Info DefaultHttpControllerSelector SelectController Admin
2017-02-20 19:36:47.6664 Info HttpControllerDescriptor CreateController
2017-02-20 19:36:47.6664 Info DefaultHttpControllerActivator Create
2017-02-20 19:36:47.6674 Info DefaultHttpControllerActivator Create Jackett.Controllers.AdminController
2017-02-20 19:36:47.6674 Info HttpControllerDescriptor CreateController Jackett.Controllers.AdminController
2017-02-20 19:36:47.6674 Info AdminController ExecuteAsync
2017-02-20 19:36:47.6674 Info ApiControllerActionSelector SelectAction
2017-02-20 19:36:47.6674 Info ApiControllerActionSelector SelectAction Selected action 'Configure()'
2017-02-20 19:36:47.6684 Info JackettAuthorizedAttribute OnAuthorizationAsync
2017-02-20 19:36:47.6684 Info JackettAuthorizedAttribute OnAuthorizationAsync
2017-02-20 19:36:47.6684 Info HttpActionBinding ExecuteBindingAsync
2017-02-20 19:36:47.6684 Info HttpActionBinding ExecuteBindingAsync
2017-02-20 19:36:47.6684 Info JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Configure()'
2017-02-20 19:36:47.6694 Info JackettAPINoCacheAttribute OnActionExecutingAsync
2017-02-20 19:36:47.6694 Info ApiControllerActionInvoker InvokeActionAsync Action='Configure()'
2017-02-20 19:36:47.6694 Info ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Configure()'
2017-02-20 19:36:47.6694 Debug IWebClient.GetString(Url:http://www.shazbat.tv/login)
2017-02-20 19:36:48.1649 Debug IWebClient: Returning Redirect => https://www.shazbat.tv/login
2017-02-20 19:36:48.1649 Debug updating Cookies  => __cfduid=d78c87c13ae3f0ab9a34c54b36f4096f21487648207; PHPSESSID=9bkcdialehdhdfukn529fsaij7
2017-02-20 19:36:48.1656 Debug IWebClient.GetString(Url:http://www.shazbat.tv/login)
2017-02-20 19:36:48.1656 Debug UnixLibCurlWebClient: Posting referer=login&query=&tv_login=######&tv_password=######&email=
2017-02-20 19:36:48.5960 Debug IWebClient: Returning Redirect => https://www.shazbat.tv/login
2017-02-20 19:36:48.5968 Debug IWebClient.GetBytes(Url:https://www.shazbat.tv/login)
2017-02-20 19:36:49.5006 Debug IWebClient: Returning OK => 6361 bytes
2017-02-20 19:36:49.5036 Info ReflectedHttpActionDescriptor ExecuteAsync Action returned 'System.Web.Http.Results.JsonResult`1[Newtonsoft.Json.Linq.JObject]'
2017-02-20 19:36:49.5044 Info ApiControllerActionInvoker InvokeActionAsync
2017-02-20 19:36:49.5044 Info JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Configure()'
2017-02-20 19:36:49.5056 Info JackettAPINoCacheAttribute OnActionExecutedAsync
2017-02-20 19:36:49.5056 Info AdminController ExecuteAsync
2017-02-20 19:36:49.5064 Info   Content-type='application/json; charset=utf-8', content-length=646
2017-02-20 19:36:49.5064 Info AdminController Dispose
2017-02-20 19:36:49.5075 Info AdminController Dispose

**Comment by kaso17** [comment]
I don't think any of the current developers has an account at Shazbat.
Please send an invite to kaso1717 -at- gmail.com and I'll fix the indexer and bring it up to date regarding newer Jackett features.

**Comment by bodenkers** [comment]
As soon as I have an invite available I will send you one. At the moment I
do not have one.

**Comment by kaso17** [comment]
Change your URL to https, that should fix it.

**Comment by Gillingham** [comment]
I've been having the same issue for months, currently with `Mono JIT compiler version 4.6.2 (Stable 4.6.2.7/08fd525 Tue Dec 27 10:49:36 UTC 2016)` and `Jackett Version 0.7.1146.0`

**Comment by kaso17** [comment]
@Gillingham please post a log

**Comment by bodenkers** [comment]
I added https and it fixed things for me

**Comment by Gillingham** [comment]
```
03-12 10:16:46 Info   http://192.168.1.56:9117/Admin/configure_indexer
System.Web.Http.Request:   http://192.168.1.56:9117/Admin/configure_indexer
03-12 10:16:46 Info WebAPIRequestLogger SendAsync
System.Web.Http.MessageHandlers: WebAPIRequestLogger SendAsync
{"indexer":"shazbat","name":"Shazbat","config":[{"id":"sitelink","value":"https://www.shazbat.tv/"},{"id":"username","value":"myuser"},{"id"     :"password","value":"|||%%PREVJACKPASSWD%%|||"},{"id":"rsskey","value":""},{"id":"cookieheader","value":""},{"id":"lasterror","value":""}]}
03-12 10:16:46 Debug POST: http://192.168.1.56:9117/Admin/configure_indexer
03-12 10:16:46 Debug Body: {"indexer":"shazbat","name":"Shazbat","config":[{"id":"sitelink","value":"https://www.shazbat.tv/"},{"id":"username",     "value":"myuser"},{"id":"password","value":"|||%%PREVJACKPASSWD%%|||"},{"id":"rsskey","value":""},{"id":"cookieheader","value":""},{"id":"la     sterror","value":""}]}
03-12 10:16:46 Info DefaultHttpControllerSelector SelectController Route='MS_SubRoutes:System.Web.Http.Routing.IHttpRouteData[]'
System.Web.Http.Controllers: DefaultHttpControllerSelector SelectController Route='MS_SubRoutes:System.Web.Http.Routing.IHttpRouteData[]'
03-12 10:16:46 Info DefaultHttpControllerSelector SelectController Admin
System.Web.Http.Controllers: DefaultHttpControllerSelector SelectController Admin
03-12 10:16:46 Info HttpControllerDescriptor CreateController
System.Web.Http.Controllers: HttpControllerDescriptor CreateController
03-12 10:16:46 Info DefaultHttpControllerActivator Create
System.Web.Http.Controllers: DefaultHttpControllerActivator Create
03-12 10:16:46 Info DefaultHttpControllerActivator Create Jackett.Controllers.AdminController
System.Web.Http.Controllers: DefaultHttpControllerActivator Create Jackett.Controllers.AdminController
03-12 10:16:46 Info HttpControllerDescriptor CreateController Jackett.Controllers.AdminController
System.Web.Http.Controllers: HttpControllerDescriptor CreateController Jackett.Controllers.AdminController
03-12 10:16:46 Info AdminController ExecuteAsync
System.Web.Http.Controllers: AdminController ExecuteAsync
03-12 10:16:46 Info ApiControllerActionSelector SelectAction
System.Web.Http.Action: ApiControllerActionSelector SelectAction
03-12 10:16:46 Info ApiControllerActionSelector SelectAction Selected action 'Configure()'
System.Web.Http.Action: ApiControllerActionSelector SelectAction Selected action 'Configure()'
03-12 10:16:46 Info JackettAuthorizedAttribute OnAuthorizationAsync
System.Web.Http.Filters: JackettAuthorizedAttribute OnAuthorizationAsync
03-12 10:16:46 Info JackettAuthorizedAttribute OnAuthorizationAsync
System.Web.Http.Filters: JackettAuthorizedAttribute OnAuthorizationAsync
03-12 10:16:46 Info HttpActionBinding ExecuteBindingAsync
System.Web.Http.ModelBinding: HttpActionBinding ExecuteBindingAsync
03-12 10:16:46 Info HttpActionBinding ExecuteBindingAsync
System.Web.Http.ModelBinding: HttpActionBinding ExecuteBindingAsync
03-12 10:16:46 Info JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Configure()'
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutingAsync Action filter for 'Configure()'
03-12 10:16:46 Info JackettAPINoCacheAttribute OnActionExecutingAsync
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutingAsync
03-12 10:16:46 Info ApiControllerActionInvoker InvokeActionAsync Action='Configure()'
System.Web.Http.Action: ApiControllerActionInvoker InvokeActionAsync Action='Configure()'
03-12 10:16:46 Info ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Configure()'
03-12 10:16:46 Info ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Configure()'
System.Web.Http.Action: ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Configure()'
03-12 10:16:46 Debug IWebClient.GetString(Url:https://www.shazbat.tv/login)
03-12 10:16:46 Error got NotImplemented/InternalServerError
03-12 10:16:46 Error request.Method: GET
03-12 10:16:46 Error request.Url: https://www.shazbat.tv/login
03-12 10:16:46 Error request.Cookies:
03-12 10:16:46 Error request.Referer:
03-12 10:16:46 Error request.RawPOSTDdata:
03-12 10:16:46 Error cookies:
03-12 10:16:46 Error headerString:

03-12 10:16:46 Debug IWebClient: Returning NotImplemented =>
03-12 10:16:46 Debug IWebClient.GetString(Url:https://www.shazbat.tv/login)
03-12 10:16:46 Debug UnixLibCurlWebClient: Posting referer=login&query=&tv_login=myuser&tv_password=*hidden*&email=
03-12 10:16:47 Error got NotImplemented/InternalServerError
03-12 10:16:47 Error request.Method: POST
03-12 10:16:47 Error request.Url: https://www.shazbat.tv/login
03-12 10:16:47 Error request.Cookies:
03-12 10:16:47 Error request.Referer: https://www.shazbat.tv/login
03-12 10:16:47 Error request.RawPOSTDdata:
03-12 10:16:47 Error cookies:
03-12 10:16:47 Error headerString:

03-12 10:16:47 Debug IWebClient: Returning NotImplemented =>
03-12 10:16:47 Info ReflectedHttpActionDescriptor ExecuteAsync Action returned 'System.Web.Http.Results.JsonResult`1[Newtonsoft.Json.Linq.JObject]'
System.Web.Http.Action: ReflectedHttpActionDescriptor ExecuteAsync Action returned 'System.Web.Http.Results.JsonResult`1[Newtonsoft.Json.Linq.JObject]'
03-12 10:16:47 Info ApiControllerActionInvoker InvokeActionAsync
System.Web.Http.Action: ApiControllerActionInvoker InvokeActionAsync
03-12 10:16:47 Info JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Configure()'
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutedAsync Action filter for 'Configure()'
03-12 10:16:47 Info JackettAPINoCacheAttribute OnActionExecutedAsync
System.Web.Http.Filters: JackettAPINoCacheAttribute OnActionExecutedAsync
03-12 10:16:47 Info AdminController ExecuteAsync
System.Web.Http.Controllers: AdminController ExecuteAsync
{"name":"Shazbat","result":"error","error":"The username and password entered do not match.","config":[{"id":"sitelink","type":"inputstring","name":"Site Link","value":"https://www.shazbat.tv/"},{"id":"username","type":"inputstring","name":"Username","value":"myuser"},{"id":"password","type":"inputstring","name":"Password","value":"hidden"},{"id":"rsskey","type":"hiddendata","name":"RSSKey","value":""},{"id":"cookieheader","type":"hiddendata","name":"CookieHeader","value":""},{"id":"lasterror","type":"hiddendata","name":"LastError","value":""}]}
03-12 10:16:47 Debug Response: {"name":"Shazbat","result":"error","error":"The username and password entered do not match.","config":[{"id":"sitelink","type":"inputstring","name":"Site Link","value":"https://www.shazbat.tv/"},{"id":"username","type":"inputstring","name":"Username","value":"myuser"},{"id":"password","type":"inputstring","name":"Password","value":"hidden"},{"id":"rsskey","type":"hiddendata","name":"RSSKey","value":""},{"id":"cookieheader","type":"hiddendata","name":"CookieHeader","value":""},{"id":"lasterror","type":"hiddendata","name":"LastError","value":""}]}
03-12 10:16:47 Info WebAPIRequestLogger SendAsync
System.Web.Http.MessageHandlers: WebAPIRequestLogger SendAsync
03-12 10:16:47 Info   Content-type='application/json; charset=utf-8', content-length=565
System.Web.Http.Request:   Content-type='application/json; charset=utf-8', content-length=565
03-12 10:16:47 Info AdminController Dispose
System.Web.Http.Controllers: AdminController Dispose
03-12 10:16:47 Info AdminController Dispose
System.Web.Http.Controllers: AdminController Dispose
```

**Comment by kaso17** [comment]
Please post the output of 
`curl -v https://www.shazbat.tv/`
It's probably a certificate validation issue.
You can try running jackett without validation ("-t true" argument) or upgrade to mono 4.8

**Comment by Gillingham** [comment]
I was running with `-t -l`, I restarted it with `-t true -l` with no change. You are correct about the curl error, `curl: (35) error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure`

Is there any possible alternative to upgrading mono? This is on a freebsd system so hand compiling to get to 4.8 would be my last resort.

**Comment by kaso17** [comment]
Sorry, the command line argument to disable certificate logging is `--IgnoreSslErrors true`.
But based on the curl error it won't fix the problem. You probably are still using openssl 0.9.8 which is to old for shazbat. See the handshake simulation on https://www.ssllabs.com/ssltest/analyze.html?d=www.shazbat.tv&s=2400%3acb00%3a2048%3a1%3a0%3a0%3a681c%3ad1a

**Comment by Gillingham** [comment]
Looks like I actually solved this, upgrade the jail to FreeBSD 10.3 which includes openssl 1.0.1, and libcurl compiled against that, and shazbat is working. Thanks for the support.
